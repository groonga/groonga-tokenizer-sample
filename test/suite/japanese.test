register tokenizers/sample

tokenize TokenSample "あいう え お"
