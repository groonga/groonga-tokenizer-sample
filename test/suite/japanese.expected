register tokenizers/sample
[[0,0.0,0.0],true]
tokenize TokenSample "あいう え お"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "あ",
      "position": 0
    },
    {
      "value": "い",
      "position": 1
    },
    {
      "value": "う",
      "position": 2
    },
    {
      "value": " ",
      "position": 3
    },
    {
      "value": "え",
      "position": 4
    },
    {
      "value": " ",
      "position": 5
    },
    {
      "value": "お",
      "position": 6
    }
  ]
]
